docs/local-development.md

```diff
@@ -0,0 +1,146 @@
+# Local Development Setup
+
+## Prerequisites
+
+- Python 3.11+ (project requires >= 3.11)
+- Node.js 22+
+- npm
+
+## Installation
+
+### 1. Clone Repository
+
+```bash
+git clone <repository-url>
+cd mcp-getgather
+```
+
+### 2. Install Dependencies
+
+#### Python Dependencies
+
+```bash
+# Using uv (recommended)
+uv sync
+
+# Or using pip
+pip install -e .
+```
+
+#### Node.js Dependencies
+
+```bash
+npm install
+```
+
+## Running the Application
+
+### Development Mode
+
+Run both frontend and backend:
+
+```bash
+npm run dev
```

Comment:
This still fails for me, not sure why.

How to reproduce:
1. Switch to this branch
2. Run `uv sync` and `npm install`
3. Run `npm run dev`

The error:
```bash
[0]   File "/Users/ariya/opensource/gather/mcp-getgather/getgather/api/main.py", line 54, in <module>
[0]     app.mount("/assets", StaticFiles(directory=BUILD_ASSETS_DIR), name="assets")
[0]                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
My bad. Before running dev, there should be some stub under api/frontend that’s generated from the Vite build. I’ve fixed it by adding a predev script to generate the stub.